src/components/Votes/Votes.tsx

```diff
@@ -24,6 +26,10 @@ export const Votes = (props: VotesProps) => {
     store.dispatch(ActionFactory.deleteVote(props.noteId));
   };
 
+  const votes = props.votes.filter((vote) => vote.user === Parse.User.current()?.id);
```

Comment:
Why do you need to filter the votes here? You've already filtered them in the Notes component with `props.voteConfiguration?.showVotesOfOtherUsers` and `activeVoting`

Reply:
We need to filter here again to avoid showing the add vote button if he can't vote again.